xamarin/Xamarin.Forms
# Update LinearGradientBrush based on FlowDirection

### Description of Change ###

Update LinearGradientBrush based on FlowDirection .

_Work in progress_

### Issues Resolved ### 

- fixes #11603 

### API Changes ###
 
 None

### Platforms Affected ### 

- Core/XAML (all platforms)
- iOS
- macOS
- Android
- UWP

### Behavioral/Visual Changes ###

FlowDirection defines not only how text flows in a textual element but also the flow direction of almost every other UI element. With this changes, changing the FlowDirection also affects to a LinearGradientBrush.

### Before/After Screenshots ### 

![brush-flowdirection-ios](https://user-images.githubusercontent.com/6755973/90035861-9d5b6480-dcc2-11ea-9ac8-7df97d2c63c1.gif)

![flow-brush-macos](https://user-images.githubusercontent.com/6755973/90122680-f1675700-dd5d-11ea-872f-a3f6c75fec63.gif)

### Testing Procedure ###
Launch Core Gallery and navigate to the brushes gallery. Select the "" and apply FlowDirection changes to see the behavior.

### PR Checklist ###
